src/components/card/card.spec.js

```diff
@@ -158,6 +158,14 @@ describe('Card', () => {
       }, wrapper.toJSON());
     });
   });
+
+  describe('when the "draggable" prop is set to true', () => {
```

Comment:
Incorrect `describe` block(draggable is not set to true in the test), the `it` block is sufficient.
Also needs a negative test for when data role is not provided.